### ISSUE TITLE
Feature Request: Write single register (0x06)

### ISSUE THREAD

**Issue by TDArlt** [SciFiDryer/ModbusMechanic#7]
Thanks for this tool which is really helpful in simulating and explaining Modbus communication, in my case for configuring filters in OT-firewalls.

For illustration purposes, it would come in handy to also have the possibility to write a single register directly (0x06).

As I don't know about all the dependencies inside your project, I don't want to fork this right away. But if you can tell me about the positions where this might have an impact, I can write these few Java lines by myself as well.

(btw. I'll fix the German translation as well then)

**Comment by SciFiDryer** [comment]
Hello, thanks for the suggestion. When I did the GUI I wasn't sure how often the read/write single register commands would be used, so I did not implement them. I can add them in easy enough. The modifications would mostly be in PacketFrame, fireSelectionEvent would need to be updated to grey out certain options, and in ModbusMechanic the generateModbusTCPRequest method would need to be updated to accept read/write single register. Keep an eye on my commits, not sure how much time I'll have in the next 2 weeks but I might be able to get a portion started at least.
